resources/popup.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
```

Comment:
(jQuery Documentation)[https://learn.jquery.com/using-jquery-core/document-ready/] shows that using this line the script is activated when the document is loaded. This way the script can be placed in the header of the html page